src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -85,23 +85,30 @@ const ProfileHeader = ({
     'medium',
   );
 
+  const classes = {
+    wrapper: [...wrapperClasses, ...wrapperClassesMedium].join(' '),
+    innerWrapper: [
+      ...innerWrapperClasses,
+      ...innerWrapperClassesMedium,
+      'usa-grid',
+      'usa-grid-full',
+    ].join(' '),
+    serviceBadge: [
+      ...serviceBadgeClasses,
+      ...serviceBadgeClassesMedium,
+      'usa-width-one-fourth',
+    ].join(' '),
+    title: [...titleClasses, ...titleClassesMedium].join(' '),
+    fullName: [...fullNameClasses, ...fullNameClassesMedium].join(' '),
+    latestBranch: [...latestBranchClasses, ...latestBranchClassesMedium].join(
+      ' ',
+    ),
+  };
+
   return (
-    <div className={[...wrapperClasses, ...wrapperClassesMedium].join(' ')}>
-      <div
-        className={[
-          ...innerWrapperClasses,
-          ...innerWrapperClassesMedium,
-          'usa-grid',
-          'usa-grid-full',
-        ].join(' ')}
-      >
-        <div
-          className={[
-            ...serviceBadgeClasses,
-            ...serviceBadgeClassesMedium,
-            'usa-width-one-fourth',
-          ].join(' ')}
-        >
+    <div className={classes.wrapper}>
+      <div className={classes.innerWrapper}>
+        <div className={classes.serviceBadge}>
```

Comment:
I'm _pretty_ sure that we can't get away from this mild case of divitis. They are mostly there to handle layout concerns.